evergreen/archive-evergreen.sh

```diff
@@ -18,12 +21,17 @@ cd -
 # working directory to "ARCHIVE_PATH" is a workaround.
 ARCHIVE_PATH=archive
 # Example archive file name: envoy-serverless-1.21.3-4-gdbdcfa34cf.rhel7.amd64.tar.gz
-ARCHIVE_DIR=envoy-serverless-${VERSION}.rhel7.amd64
+ARCHIVE_DIR=envoy-serverless-${VERSION}.${DISTRO}.${ARCH}
 mkdir -p ${ARCHIVE_PATH}/${ARCHIVE_DIR}
 
+if [ $ARCH == "aarch64" ]; then
+	OUTPUT_DIR="aarch64"
+else
+	OUTPUT_DIR="k8"
+fi
 # Only include the envoy binary and the hot-restarter. The start script and the static config should
 # be prepared by the agent.
-cp build/execroot/envoy/bazel-out/k8-opt/bin/source/exe/envoy-static ${ARCHIVE_PATH}/${ARCHIVE_DIR}/envoy-serverless
+cp build/execroot/envoy/bazel-out/$OUTPUT_DIR-opt/bin/source/exe/envoy-static ${ARCHIVE_PATH}/${ARCHIVE_DIR}/envoy-serverless
```

Comment:
I feel like the OUTPUT_DIRs should be "aarch64-opt" and "k8-opt".  "k8" is technically not the name of a directory, but the variable name implies that it is.